### ISSUE TITLE
mdChips: error when removing chips with autocomplete

### ISSUE THREAD

**Issue by dephiros** [angular/material#10036]
**Actual Behavior**:
 -  `What is the issue? *` error when removing chips with autocomplete
 -  `What is the expected behavior?` 

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:` https://material.angularjs.org/latest/demo/chips
 -  `Details:` 
    - under contact chips demo and custom input demo with autocomplete
    - Add one or more chip
    - remove the chip by clicking on 'x'. Error should show in console log

**Angular Versions**: *
 -  `Angular Version:` 1.5.5
 -  `Angular Material Version:` 1.1.1

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` 54.0.2840.98 (64-bit)
 -  `OS: *` Mac
 -  `Stack Traces:`
  ```
angular.js:13550 TypeError: Cannot read property 'nextTick' of undefined
    at e.removeChipAndFocusInput (angular-material.min.js:14)
    at fn (eval at compile (angular.js:14432), <anonymous>:4:516)
    at b (angular.js:15485)
    at e (angular.js:25018)
    at b.$eval (angular.js:17229)
    at b.$apply (angular.js:17329)
    at HTMLButtonElement.<anonymous> (angular.js:25023)
    at Qf (angular.js:3456)
    at HTMLButtonElement.d (angular.js:3444)
  ```

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by tomchambers2** [comment]
Probably relevant: #9520

**Comment by dephiros** [comment]
my bad for creating a duplicate issue but woohoo the issue will be fixed in 1.1.2. Happy Friday!
